app/css/views/services-list.css

```diff
@@ -6,10 +6,28 @@
   background-image: url('services-list/ip-camera.svg');
 }
 
-[data-icon="motion-sensor"] {
+[data-icon="door-lock"] {
+  background-image: url('services-list/door-lock.svg');
+}
+
+@keyframes motion-sensor-animation {
+  0%, 90% {
+    background-size: 2.7rem;
+  }
+
+  5%, 95% {
+    background-size: 2.3rem;
+  }
+
+  10%, 85%, 100% {
+    background-size: 2.5rem;
+  }
+}
+
+.motion-sensor-item {
   background-image: url('services-list/motion-sensor.svg');
 }
 
-[data-icon="door-lock"] {
-  background-image: url('services-list/door-lock.svg');
+.motion-sensor-item--motion-detected {
```

Comment:
Yes, I prefer not to spend too much time on the design/UX while the scope of the project is still not defined.